apps/aehttp/priv/oas3.yaml

```diff
@@ -543,6 +543,7 @@ paths:
           content:
             application/json:
               schema:
+                $ref: "#/components/schemas/Error"
```

Comment:
was fixed in https://github.com/aeternity/aeternity/commit/684e89ffe45e3f15a7e50bb71773951a4a9d9b83#diff-c611e73074019dbed160a86754e436f5db05174f8e5f80f361ccbc4dc465f3a6R546